spaceship/lib/spaceship/tunes/device_type.rb

```diff
@@ -1,39 +1,16 @@
+require_relative "display_family"
+
 module Spaceship
   module Tunes
     # identifiers of devices that App Store Connect accepts screenshots for
     class DeviceType
-      @types = [
-        # iPhone
-        'iphone35',
-        'iphone4',
-        'iphone6', # 4.7-inch Display
-        'iphone6Plus', # 5.5-inch Display
-        'iphone58', # iPhone XS
-        'iphone65', # iPhone XS Max
-
-        # iPad
-        'ipad', # 9.7-inch Display
-        'ipad105',
-        'ipadPro',
-        'ipadPro11',
-        'ipadPro129',
-
-        # Apple Watch
-        'watch', # series 3
-        'watchSeries4',
-
-        # Apple TV
-        'appleTV',
-
-        # Mac
-        'desktop'
-      ]
-      class << self
-        attr_accessor :types
+      def self.types
+        warn("Spaceship::Tunes::DeviceType has been deprecated. Use the new Spaceship::Tunes::DisplayFamily class instead.")
```

Comment:
I dont think we have a precedent for `warn` in spaceship yet. I think it's a good use of the method.
Could you add either a comment with a URL to this PR, or add the URL to the warn message?

Reply:
Sure, I updated the message to link to the PR 🙂

Reply:
Sure, I updated the message to link to the PR 🙂